packages/now-routing-utils/test/superstatic.spec.js

```diff
@@ -88,6 +88,42 @@ test('convertCleanUrls true', () => {
   assertRegexMatches(actual, mustMatch, mustNotMatch);
 });
 
+test('convertCleanUrls true, trailingSlash true', () => {
+  const actual = convertCleanUrls(true, true);
+  const expected = [
+    {
+      src: '^/(?:(.+)/)?index(?:\\.html)?/?$',
+      headers: { Location: '/$1/' },
+      status: 301,
+    },
+    {
+      src: '^/(.*)\\.html/?$',
+      headers: { Location: '/$1/' },
+      status: 301,
+    },
+  ];
+  deepEqual(actual, expected);
+
+  const mustMatch = [
```

Comment:
It should match both with and without the trailing slash

Reply:
True. Should I just copy/pasta and double the number of matches here?

Reply:
SGTM

Reply:
Done 👍